pages/projects/index.page.js

```diff
@@ -1,5 +1,47 @@
+import axios from "axios";
+import { useQuery } from "@tanstack/react-query";
+import { getFormattedDate } from "../../utils/date";
+
+import { Button, ProgressBar } from "../../components";
+import { Card, Container, Text } from "./projects.styles";
+
 const ProjectsList = () => {
-  return <div>Project List</div>;
+  const { data: campaigns, isLoading } = useQuery({
+    queryKey: ["campaigns"],
```

Comment:
Suggestion: create `queries.js` that exports constants with all the query keys to avoid magic strings